edge/pkg/edgehub/level_edge.go

```diff
@@ -0,0 +1,154 @@
+package edgehub
+
+import (
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+	"k8s.io/klog/v2"
+	"math"
```

Comment:
Sort the import. Built-in packages in Go should be placed separately in the first part.